### PR TITLE
Update dbcc-shrinkfile-transact-sql.md

### PR DESCRIPTION
**Pull request by hdharshana**
Updated Syntax for WAIT_AT_LOW_PRIORITY. Included the parameter MAX_DURATION as that appears to be working for RTM release of SQL 2022.

### CODE REVIEW

**Comment by jborsecnik** [comment]
@WilliamDAssafMSFT, 

Can you review the proposed changes? ***Note: Unless you are confident that the includes are exactly correct, these changes should be done in the private repo so we can view them on staging.***

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/t-sql/database-console-commands/dbcc-shrinkfile-transact-sql.md [hunk 1]
```diff
@@ -111,16 +117,24 @@ The wait at low priority feature reduces lock contention. For more information,
...
+This is an optional parameter. The default value when not specified = 1 minute. 
 
-If a new shrink operation in WAIT_AT_LOW_PRIORITY mode cannot obtain a lock due to a long-running query, the shrink operation will eventually timeout after 60,000 milliseconds (1 minute) and will silently exit. This will occur if the shrink operation cannot obtain the Sch-M lock due to concurrent query or queries holding Sch-S locks. When a timeout occurs, an error 49516 message will be sent to the SQL Server error log, for example: `Msg 49516, Level 16, State 1, Line 134 Shrink timeout waiting to acquire schema modify lock in WLP mode to process IAM pageID 1:2865 on database ID 5`. At this point, you can simply retry the shrink operation in WAIT_AT_LOW_PRIORITY mode knowing that there would be no impact to the application.
+It is the wait time (an integer value specified in minutes) that the shrink operation Sch-M lock request will wait with low priority when executing the command. If the operation is blocked for the duration, the specified ABORT_AFTER_WAIT action will be executed. MAX_DURATION time is always in minutes, and the word MINUTES can be omitted.
```
Comment (WilliamDAssafMSFT) [inline docs/t-sql/database-console-commands/dbcc-shrinkfile-transact-sql.md]:
```suggestion
The shrink operation's Sch-M lock request will wait with low priority when executing the command for the MAX_DURATION in minutes. If the operation is blocked for the duration, the specified ABORT_AFTER_WAIT action will be executed. MAX_DURATION time is always in minutes, and the word MINUTES can be omitted. 

```

**Review by WilliamDAssafMSFT — CHANGES_REQUESTED** [review]

**Review by WilliamDAssafMSFT — APPROVED** [review]

**Comment by WilliamDAssafMSFT** [comment]
#sign-off

**Comment by WilliamDAssafMSFT** [comment]
Bringing this PR from the public PR into the private repo, where it needs to be.

**Pull request closed without merging**